mews-operations/availabilityBlock.md

```diff
@@ -33,4 +33,178 @@
+            "email": "angelina.jolie@10001-438-sample-portfolio-hotel-10001.com",
+            "firstName": "Angelina",
+            "lastName": "Jolie",
+            "title": null,
+            "telephone": null,
+            "nationalityCode": null,
+            "languageCode": null,
+            "loyaltyCode": null,
+            "address": null
+        },
+        "company": null,
+        "notes": "This is a note."
+    }  
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `clientToken` | `string` | required | Client token of the channel manager. |
+| `connectionToken` | `string` | required | Token of a concrete connection. |
+| `messageId` | `string` | required | Unique identification of the message. Used for asynchronous confirmations. |
+| `availabilityBlock` | [`Availability block`](#availability-block) object | required | Availablity block data. |
+
+
+#### Availability block
+| Property | Type | Contract| Description |
+| :-- | :-- | :-- | :-- |
+| `code`| `string`| required | Unique reference code from external system for the availability block. Will be returned after confirmation containing this code. |
+| `confirmationNumber` | `string` | optional | Mews confirmation number for the block. If updating or deleting an existing block, this property is required. |
+| `status` | [`Status`](#status) object| required | State of the availability block. |
+| `name` | `string` | required | Name of the availability block. |
+| `dates` | [`Dates`](#dates) object | required| Dates of the availability block. |
+| `releaseStrategy` | [`Release strategy`](#release-strategy) object | required | Release strategy of the availability block. |
+| `originalRateCode` | `string` | required | Original rated code of the availability block. |
+| `spaceCategories` | [`Space category allocation`](#space-category-allocation) collection | required | Allocated categories of the availability block. |
+| `booker` | [`Customer`](./reservations.md#customer) object | required | The main booker. This does not mean that the person has arrived at the property. |
+| `company` | [`Company`](./reservations.md#company) object | optional | The company associated with the availability block. |
+| `notes` | `string` | optional | Notes related to the availability block. |
+
+
+#### Dates
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `start` | `string` | required | Start date in format `yyyy-mm-dd`, e.g. `2024-12-05`, this is the first day of the block and so can be booked for stay. |
+| `end` | `string` | required | End date in format `yyyy-mm-dd`, e.g. `2024-12-08`, this day is _not_ included in the block and so cannot be booked for stay (it is analogous to reservation departure). |
+
+
+#### Release strategy
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `type` | `string` | [`Release strategy type`](#release-strategy-type) required | release strategy for the availability block. |
```

Comment:
The link to the string enumeration should be in the **type** column not the **required** column.
```suggestion
| `type` | `string` [Release strategy type](#release-strategy-type)  | required | release strategy for the availability block. |
```